litmus-portal/frontend/src/components/Unimodal/index.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Modal from '@material-ui/core/Modal';
+import Button from '@material-ui/core/Button';
+import useStyles from './styles';
+
+/* DelUser, NewUserModal, ResetModal need to be shifted */
+
+interface UnimodalProps {
+  children?: any;
```

Comment:
Don't keep the type as any, instead replace it with a `React.ReactNode` since the children component would be a potential DOM node

Reply:
I was actually confused about which one to use, `JSX.Element` or `ReactNode`. So I followed the advice [here](https://github.com/Microsoft/TypeScript/issues/6471). Will change this to `ReactNode`.